test/backend/lib.api.streamer.test.js

```diff
@@ -57,6 +57,6 @@ describe('Streamer', function() {
     });
   });
 
-  after(utils.wipeModels([Report]));
-  after(utils.expectModelsEmpty);
+    after(utils.wipeModels([Report]));
+    after(utils.expectModelsEmpty);
```

Comment:
Trivial but I think the spacing should be in the previous location to align with the before logic. Thoughts?

Reply:
Sure. Fixed 👍